kotlin-css/src/commonMain/kotlin/kotlinx/css/CssBuilder.kt

```diff
@@ -116,6 +119,10 @@ interface CssBuilder : StyledElement, RuleContainer {
         val selectorString = selector.joinToString { "> $it" }
         return selectorString(block)
     }
+    fun child(vararg selector: String, css: CssBuilder): Rule {
```

Comment:
If this is needed, then what about all other methods?

Reply:
It is only needed for `compareTo` method, which uses other rule (already converted to CssBuilder because we don't store lambdas)